src/session_impl.cpp

```diff
@@ -6865,6 +6950,7 @@ namespace aux {
 			, struct tracker_response const& resp)
 		{
 			TORRENT_UNUSED(tracker_ips);
+			if (!should_log()) return;
```

Comment:
This is the code
```java
#ifndef TORRENT_DISABLE_LOGGING
			debug_log("==> TRACKER REQUEST \"%s\" event: %s abort: %d"
				, req.url.c_str()
				, (req.event==tracker_request::stopped?"stopped"
					:req.event==tracker_request::started?"started":"")
				, m_abort);

			// if we're not logging session logs, don't bother creating an
			// observer object just for logging
			if (m_abort && alerts().should_post<log_alert>())
			{
				std::shared_ptr<aux::tracker_logger> tl(new aux::tracker_logger(m_ses));
				m_ses.queue_tracker_request(req, tl);
			}
			else
#endif
			{
				m_ses.queue_tracker_request(req, shared_from_this());
			}
```
I see you are right, but I'm not quite sure if you want something specific